core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -112,6 +116,8 @@ public AbstractClient(ClientContext context, InetSocketAddress address,
     mRetryPolicySupplier = retryPolicySupplier;
     mServiceVersion = Constants.UNKNOWN_SERVICE_VERSION;
     mRpcThreshold = mContext.getClusterConf().getMs(PropertyKey.USER_LOGGING_THRESHOLD);
+    mRpcCallDeadlineDurationMs =
```

Comment:
@ZhuTopher For 1;
Here's where that value `("30sec")` is converted to a `long` in milliseconds.